examples/AWSDriverExample/src/main/java/software/amazon/ReadWriteSplittingMySQLExample.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon;
+
+import com.zaxxer.hikari.HikariConfig;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import software.amazon.jdbc.HostSpec;
+import software.amazon.jdbc.PropertyDefinition;
+import software.amazon.jdbc.plugin.failover.FailoverFailedSQLException;
+import software.amazon.jdbc.plugin.failover.FailoverSuccessSQLException;
+import software.amazon.jdbc.plugin.failover.TransactionStateUnknownSQLException;
+
+
+public class ReadWriteSplittingMySQLExample {
+
+  // User configures connection properties here
+  public static final String MYSQL_CONNECTION_STRING =
+      "jdbc:aws-wrapper:mysql://test-db.cluster-XYZ.us-east-2.rds.amazonaws.com:5432/readWriteSplittingExample";
+  private static final String USERNAME = "username";
+  private static final String PASSWORD = "password";
+
+  public static void main(String[] args) throws SQLException {
+
+    final Properties props = new Properties();
+
+    // Enable readWriteSplitting, failover, and efm plugins and set properties
+    props.setProperty(PropertyDefinition.PLUGINS.name, "readWriteSplitting,failover,efm");
+    props.setProperty(PropertyDefinition.USER.name, USERNAME);
+    props.setProperty(PropertyDefinition.PASSWORD.name, PASSWORD);
+
+    /**
+     * Optional: configure read-write splitting to use internal connection pools (the getPoolKey
+     * parameter is optional, see UsingTheReadWriteSplittingPlugin.md for more info).
```

Comment:
Can we use  link to a md file?

Reply:
I am not sure if we should leave an absolute link in case the file does get moved to other locations.